<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputWidget</class>
 <widget class="QWidget" name="OutputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="outputBrowseButton">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="outputEdit">
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Output file...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="presetsFilterBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>All Presets</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Internal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>User</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="presetsBox">
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Load...</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Save...</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="formatsFilterBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>All Formats</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Video &amp; Audio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Images</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Audio Only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Video Only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Special</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="formatsBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QToolButton" name="addBlockButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="videoWidget">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QCheckBox" name="noVideoButton">
          <property name="text">
           <string>Video</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QRadioButton" name="videoTranscodeButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Transcode</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="videoCopyButton">
            <property name="text">
             <string>Copy Stream</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="audioWidget">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QCheckBox" name="noAudioButton">
          <property name="text">
           <string>Audio</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QRadioButton" name="audioTranscodeButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Transcode</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="audioCopyButton">
            <property name="text">
             <string>Copy Stream</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>333</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="blocksWidget" native="true">
         <layout class="QVBoxLayout" name="blocksLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="outputTabs">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="audioTab">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <attribute name="title">
           <string>Audio</string>
          </attribute>
          <layout class="QVBoxLayout" name="audioLayout" stretch="0,0">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QWidget" name="audioSelectionWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>3</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="mainAudioCodecWidget" native="true">
             <layout class="QFormLayout" name="formLayout_2">
              <property name="horizontalSpacing">
               <number>3</number>
              </property>
              <property name="verticalSpacing">
               <number>3</number>
              </property>
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <item row="0" column="1">
               <widget class="QWidget" name="audioCodecWidget" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="spacing">
                  <number>3</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="audioBitrateButton">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Bitrate</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="audioBitRateEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="frame">
                 <bool>false</bool>
                </property>
                <property name="suffix">
                 <string> Auto</string>
                </property>
                <property name="minimum">
                 <number>0</number>
                </property>
                <property name="maximum">
                 <number>1000</number>
                </property>
                <property name="singleStep">
                 <number>10</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="customOptionsWidget" native="true">
         <layout class="QFormLayout" name="customOptionsLayout">
          <property name="horizontalSpacing">
           <number>3</number>
          </property>
          <property name="verticalSpacing">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="addParam">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Custom option...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionResize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Resize</string>
   </property>
   <property name="toolTip">
    <string>Resize the video</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionFrameRate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Frame rate</string>
   </property>
   <property name="toolTip">
    <string>Change the frame rate of the video</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionVideoCodec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Video codec</string>
   </property>
   <property name="toolTip">
    <string>Select the video codec</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="actionVideoBitrate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Video quality</string>
   </property>
   <property name="toolTip">
    <string>Options for the quality of the video (bitrate...)</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="actionProfile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Video codec profile</string>
   </property>
   <property name="toolTip">
    <string>Set the profile of the video codec</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionLoops">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Loops</string>
   </property>
   <property name="toolTip">
    <string>Set the number of loops for the video</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionStartNumber">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sequence start number</string>
   </property>
   <property name="toolTip">
    <string>Set the number of the first frame of the sequence</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionAlpha">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Transparency / Alpha</string>
   </property>
   <property name="toolTip">
    <string>Alpha options</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionPixelFormat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Color management</string>
   </property>
   <property name="toolTip">
    <string>Set the color management of the video</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionSampling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Audio sampling</string>
   </property>
   <property name="toolTip">
    <string>Change audio sampling</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionAudioCodec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Audio codec</string>
   </property>
   <property name="toolTip">
    <string>Select the audio codec</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
